packages/components/src/palette-edit/test/index.js

```diff
@@ -61,3 +66,46 @@ describe( 'getNameForPosition', () => {
 		);
 	} );
 } );
+
+describe( 'PaletteEdit', () => {
+	const defaultProps = {
+		gradients: false,
+		colors: [ { color: '#ffffff', name: 'Base', slug: 'base' } ],
+		onChange: jest.fn(),
+		paletteLabel: 'Test label',
+		emptyMessage: 'Test empty message',
+		canOnlyChangeValues: true,
+		canReset: true,
+		slugPrefix: '',
+	};
+
+	it( 'opens color selector for color palettes', async () => {
```

Comment:
Thanks for updating so quickly!

The tests are passing for me locally, but I noticed the Github Action appears to be failing. I kicked it off again, but if it still fails, would it be worth trying to remove the `async` here? I wasn't sure if it's needed since the other test doesn't have it 🤔

Reply:
That was an oversight from previous testing. Thanks again for 👀

Reply:
Hrm, for some reason the test is still complaining 🤔
Happy to help debug on Monday!

Reply:
Yeah it's curious. `fireEvent` is already wrapped in `act()` apparently. 
Tried using async `getBy*` queries as well. Still failed. 🤷 
Oh well.